iglo/league/utils/paring.py

```diff
@@ -26,18 +26,65 @@ def round_robin(n: int) -> Pairing:
         return list(_round_robin_odd(d, n))
 
 
+def banded_round_robin(player_count: int, band_size: int, add_byes: bool = True) -> Pairing:
+  by_round = []
+  for player_distance in range(band_size, 0, -1):
+    round_2pd_0 = set() # round 2*player_distance
```

Comment:
maybe let's call it odd and even distances?

Reply:
brilliant :D
I worked/refactored with this algo so much that I totally missed obvious naming.